### ISSUE TITLE
tabnumbers does not display number of the last tab

### ISSUE THREAD

**Issue by brandonw** [vimperator/vimperator-labs#116]
If more than 1 tab is open, and the last tab is not currently selected, `:set tabnumbers` does not render a tab number for the final tab.

**Comment by llinfeng** [comment]
If there had been such a bug, it has been solved. 
![2015-01-22_220812](https://cloud.githubusercontent.com/assets/5890407/5869205/3a5e66cc-a283-11e4-9bbe-3ae85396518e.png)

**Comment by brandonw** [comment]
@llinfeng, try selecting a tab that is not the final tab, and see if the tab number is still displayed.

**Comment by gkatsev** [comment]
It seems to have been solved for me on my linux machine with the developer edition styles but not on OSX.

**Comment by ciuncan** [comment]
I can confirm this issue still persists in following versions (my case):
Ubuntu:  14.10
Firefox:  36.0
Vimperator: 3.8.3

**Comment by gkatsev** [comment]
@ciuncan can you try out vimperator 3.9?

**Comment by ciuncan** [comment]
I thought I had the latest version, but it seems https://addons.mozilla.org/ doesn't have the update yet. Nevermind, I upgraded to 3.9. Unfortunately problem still persists. Is there anything I could try cleaning, like some cache or so?

**Comment by gkatsev** [comment]
Yeah, AMO takes a while to get the new releases published. I guess the regular firefox skin has this issue where-as the developer skin doesn't. Interesting.

**Comment by timss** [comment]
Still able to reproduce as of latest, FF 45, Vimperator 3.13.1.
